bin/compile

```diff
@@ -81,9 +83,12 @@ cat<<\EOF > $BUILD_DIR/.profile.d/pg-path.sh
 PATH=$HOME/vendor/postgresql/bin:$PATH
 
 export PGHOST=/tmp
-export PG_DATA_DIR=$HOME/vendor/postgresql/data
+export PGDATA=$HOME/vendor/postgresql/data
 
-pg_ctl -D $PG_DATA_DIR -l .pg.log -w start
+if ! pg_ctl -D $PGDATA status >/dev/null;
```

Comment:
If we're exporting `$PGDATA` does that mean we can remove the `-D` flags from everywhere?